internal/charmstore/search_test.go

```diff
@@ -657,6 +667,24 @@ var searchTests = []struct {
 		results: Entities{
 			searchEntities["mysql"].entity,
 		},
+	}, {
+		about: "name search",
+		sp: SearchParams{
+			Text: "wordpress",
+		},
+		results: Entities{
+			searchEntities["wordpress"].entity,
+			searchEntities["wordpress-simple"].entity,
+		},
+	}, {
+		about: "case insensitive search",
+		sp: SearchParams{
+			Text: "WORDPRESS",
+		},
+		results: Entities{
+			searchEntities["wordpress"].entity,
+			searchEntities["wordpress-simple"].entity,
+		},
```

Comment:
Wasn't one of the failing cases where someone searching for "mongo" didn't find "mongodb"?